client/account-recovery/account-recovery-root/index.jsx

```diff
@@ -35,13 +41,44 @@ const getPageInfo = ( translate, step ) => {
 			trackerTitle: 'Account Recovery > Reset Password',
 			documentHeadTitle: translate( 'Reset Password ‹ Account Recovery' ),
 		},
-
+		[ STEPS.RESET_PASSWORD_EMAIL ]: {
+			trackerTitle: 'Account Recovery > Reset Password Email',
+			documentHeadTitle: translate( 'Reset Password < Email' ),
```

Comment:
:disappointed: 73 existing translations will be lost with this change.
If the string must change, the following option has already been translated **31** times:
`translate( 'Reset password' )` *ES Score: 8*
See 1 additional suggestion in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/add/account-recovery/redux-plumbing-for-new-password-form)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).